package.json

```diff
@@ -34,19 +35,18 @@
     "jest": "^24.9.0",
     "jest-cli": "^24.9.0",
     "lint-staged": "^10.0.0",
-    "lodash.get": "^4.4.2",
     "npmpub": "^5.0.0",
     "postcss": "^7.0.26",
     "postcss-scss": "^2.0.0",
     "prettier": "^1.19.1",
     "rimraf": "^3.0.0",
-    "stylelint": "^12.0.0"
+    "stylelint": "^13.0.0"
   },
   "peerDependencies": {
-    "stylelint": "^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0"
```

Comment:
Hmm... I'm still not sure if it's a good idea to drop support for everything else than the newest version.

Reply:
Well, the thing sis that

1. what was the last time you tested stylelist-scss master with stylelint 8.0.0? Or any version other than the latest?
2. if you are going to make sure every future version works with stylelint, why don't you just use `>=`?
3. This should be a major release IMO and we should try to merge #429 too, which makes it  a good time to remove this

Reply:
I can revert it BTW, my initial patch didn't change this.

Reply:
> what was the last time you tested stylelist-scss master with stylelint 8.0.0? Or any version other than the latest?

I don't think that you really understand my point of view here. I don't really care about stylelint version 8, and I don't know if it has even any users other than some outdated projects. I just don't think that it is a good idea to drop the support for 5 versions at the same time, as the chances that there are people using some those versions is really high.

I actually have one React Native project where I had to lock the stylelint version to v9.x because newer stylelint versions depend on some dependency that uses `@types/node` which messes up the Typescript typings for React Native in my project and causes Typescript to throw errors.

I've ran into similar issues with other npm packages, where I could not update to the latest version of a package because it required the newest version of a peer dependency, so the only solution is not to update.

The thing is that managing peer dependency versions is a complicated thing, and IMO the best way is to gradually drop supported versions from the end and release those in separate releases.

> if you are going to make sure every future version works with stylelint, why don't you just use `>=`?

 Because I don't know if the version that we have now works with a stylelint version that gets released two years from now. That's why the versions are whitelisted.

Reply:
Hence why I reverted that change and I just added v13.0.0 in peerDependencies.

I still believe this should be cleaned up later.

Reply:
Yes, it definitely should be cleaned up, but I need to see if there is any data about downloads per stylelint version to get an idea if people are still using those old versions.

Reply:
Not sure where you can find such metrics TBH.

Anyway, I assume this PR should be good to go as is? I need to rebase a few of my other PRs after this gets merged.

Reply:
I like the idea @XhmikosR suggested,
> if you are going to make sure every future version works with stylelint, why don't you just use >=